nixos/modules/security/pam.nix

```diff
@@ -77,6 +77,15 @@ let
         '';
       };
 
+      googleOsLoginAuth = mkOption {
+        default = config.security.pam.enableGoogleOsLogin;
```

Comment:
This does only make sense for `/etc/pam.d/sshd`, so better make that default to false and let `security.googleOSLogin.enable` enable `security.pam.services.sshd.googleOsLoginAuth`.

Reply:
This was changed.

Setting `security.googleOsLogin.enable` to true will now set
```
security.pam.services.sshd.makeHomeDir = true
security.pam.services.sshd.googleOsLoginAccountVerification = true
```

And we'll add a trigger for setting `googleOsLoginAuthentication`. This removes some oslogin specific cruft from the PAM module itself, and only configures oslogin pam for sshd.